shared/css/conversation.css

```diff
@@ -967,6 +967,31 @@ html[dir="rtl"] .text-chat-entry.received > p:after {
   align-self: auto;
 }
 
+.text-chat-notif {
+  display: flex;
+}
+
+.text-chat-notif > .content-wrapper {
+  margin-left: 10px;
```

Comment:
Note: where you've got margin-left and margin-right (below), you'll need to provide [html="rtl"] versions of these to flip the borders for rtl languages. You can test this in the ui-showcase (there's an rtl checkbox at the top of the page)